build.rs

```diff
@@ -1,12 +1,20 @@
-use chrono::Utc;
+use chrono::{DateTime, NaiveDateTime, Utc};
+use std::env;
 use std::process::Command;
 
 fn main() {
     // Generate the default 'cargo:' instruction output
     println!("cargo:rerun-if-changed=build.rs");
 
     // get timestamp
-    let now = Utc::now();
+    let now = match env::var("SOURCE_DATE_EPOCH") {
+        Ok(val) => {
+            let naive = NaiveDateTime::from_timestamp(val.parse::<i64>().unwrap(), 0);
```

Comment:
Nit: The unwrap here will cause build to exit with error so maybe error can be handled here, but since this is only used while performing build so this is looks fine and not worth a repush.

Reply:
I think unwrap is fine in the build file, I mean what do we want to do with the error anyway.